.github/CONTRIBUTING.md

```diff
@@ -36,6 +36,25 @@ This pull request will be a perfect start to find the fix :)
 
 # Working with the codebase
 
+### JDK Requirements
+
+**Important:** TestNG requires **JDK 17 or higher** to build, but the compiled artifacts target **Java 11**.
```

Comment:
I think we should upgrade from 17 to 21 as a new LTS was released few weeks ago

Reply:
Fixed